docs/194-inicjalizacja.md

```diff
@@ -0,0 +1,87 @@
+# Inicjalizacja bota
+  
+## Punkt wejściowy
```

Comment:
przeczytałem to i nic z tego nie zrozumiałem
mam wrażenie że jest za dużo informacji upchniętych na raz
i musiałbym przeczytać to z 5-10 razy, żeby ogarnąć co autor miał na myśli

Reply:
a teraz?